internal/pkg/pluginmanager/manager.go

```diff
@@ -2,16 +2,23 @@ package pluginmanager
 
 import (
 	"errors"
+	"fmt"
 	"log"
 	"os"
 	"path/filepath"
 
+	"github.com/spf13/viper"
+
 	"github.com/merico-dev/stream/internal/pkg/configloader"
 )
 
 func DownloadPlugins(conf *configloader.Config) error {
 	// create plugins dir if not exist
-	pluginsDir := filepath.Join(".", "plugins")
+	pluginsDir := viper.GetString("plugin-dir")
```

Comment:
Let's change it to pluginDir so that it's consistent.

Reply:
plugin-dir is to be consistent with config-file

![image](https://user-images.githubusercontent.com/18692628/148021121-554c6f18-5683-49be-a760-ca1dcb71f4db.png)